android/src/main/java/com/oijusti/firebaseuiauth/RNFirebaseuiAuthModule.java

```diff
@@ -70,6 +70,7 @@ public void signIn(final ReadableMap config, final Promise promise) {
     final String privacyPolicyUrl = config.hasKey("privacyPolicyUrl") ? config.getString("privacyPolicyUrl") : null;
     final boolean allowNewEmailAccounts = !config.hasKey("allowNewEmailAccounts") || config.getBoolean("allowNewEmailAccounts");
     final boolean requireDisplayName = !config.hasKey("requireDisplayName") || config.getBoolean("requireDisplayName");
+    final boolean allowAutoUpgradeAnonymousUsers = config.hasKey("allowAutoUpgradeAnonymousUsers") ? config.getBoolean("allowAutoUpgradeAnonymousUsers") : false;
```

Comment:
It seems here we can simplify it,
`    final boolean allowAutoUpgradeAnonymousUsers = config.hasKey("allowAutoUpgradeAnonymousUsers") && config.getBoolean("allowAutoUpgradeAnonymousUsers");
`